packages/frontend-2/assets/css/tailwind.css

```diff
@@ -8,6 +8,23 @@
  * Don't pollute this - it's going to be bundled in all pages!
  */
 
+@layer components {
```

Comment:
should probably go in the extracted package, but i don't have enough chops to do it